env0/data_sshkey.go

```diff
@@ -62,26 +64,28 @@ func dataSshKeyRead(ctx context.Context, d *schema.ResourceData, meta interface{
 func getSshKeyByName(name interface{}, meta interface{}) (*client.SshKey, error) {
 	apiClient := meta.(client.ApiClientInterface)
 
```

Comment:
nothing much really changed here... the same code is just inside a retry mechanism.